src/frontend/src/pages/CreateProjectPage/CreateProjectFormView.tsx

```diff
@@ -78,21 +78,20 @@ const CreateProjectFormView: React.FC<CreateProjectFormViewProps> = ({ allowSubm
     >
       <PageTitle title={'New Project'} previousPages={[{ name: 'Projects', route: routes.PROJECTS }]} />
       <PageBlock title={''}>
-        <Grid container spacing={2}>
-          <Grid item xs={12} md={3}>
-            <FormControl>
-              <FormLabel>Change Request ID</FormLabel>
+        <Grid container sx={{ marginBottom: '24px' }}>
+          <Grid item xs={12} md={6}>
+            <FormControl sx={{ width: '95%' }}>
```

Comment:
these form controls should have the fullwidth attribute and then add padding instead of having them take up less of the grid